apps/client/src/pages/LocationPage.js

```diff
@@ -40,7 +40,10 @@ const LocationPage = ({ topic }) => {
         action: `postcode - ${slug.replace("-", " ")}`,
         name: address.postalCode.substring(0, 4),
       });
-      context.address = address;
+      context.setData({
+        address,
+        data: null,
```

Comment:
I'm confused. In `defaultValues` there is `{ topic: null, address: null }`. There is no `data` prop.
Here we have a `setData(data)` where `data` has a `data` property.